README.md

```diff
@@ -26,3 +26,52 @@ full release to [PowerShell Gallery](https://www.powershellgallery.com/).
 Please check out common DSC Community [contributing guidelines](https://dsccommunity.org/guidelines/contributing).
 
 ## Resources
```

Comment:
Were you updating this for everything you submitted? Did you want to remove ClientSettings in this PR too?

Reply:
yeah I was updating it for everything.  I was still debating if we should remove that yet.  I didn't want to update it in to other repo, Right now unless we make changes to it, you can't make changes to the default client settings, client setting